src/AppBundle/Entity/ProcurationRequest.php

```diff
@@ -684,8 +702,38 @@ public function getFoundProxy(): ?ProcurationProxy
         return $this->foundProxy;
     }
 
-    public function setFoundProxy(?ProcurationProxy $procurationProxy)
+    public function setFoundProxy(?ProcurationProxy $procurationProxy): void
     {
         $this->foundProxy = $procurationProxy;
     }
+
+    public function getFoundBy(): ?Adherent
+    {
+        return $this->foundBy;
+    }
+
+    public function setFoundBy(?Adherent $foundBy): void
+    {
+        $this->foundBy = $foundBy;
+    }
+
+    public function getReminded(): int
```

Comment:
Is this getter used? I would have liked "addReminder`/`remind` instead as increase/decrease methods and kept the haser.